### ISSUE TITLE
Crash while running on x64 platform

### ISSUE THREAD

**Issue by omnight** [awestroke/Javascript.SourceMapper.NET#1]
Hi, first off, thanks for the library. I've been looking around a bit for something that would let me do sourcemap lookups in .NET and this seems to work nicely.

This is the error I am getting with version 1.0.0 (NuGet) running on x64 platform:

> System.DllNotFoundException was unhandled by user code
>   HResult=-2146233052
>   Message=Unable to load DLL 'JsSourceMapper_FFI': The specified module could not be found. (Exception from HRESULT: 0x8007007E)
>   Source=Javascript.SourceMapper
>   TypeName=""
>   StackTrace:
>        at Javascript.SourceMapper.RustFfiApi.cache_init(Byte[] json)
>        at Javascript.SourceMapper.SourceMapCache..ctor(String json) in C:\Users\orse\develop\private\Javascript.SourceMapper.NET\Javascript.SourceMapper\SourceMapCache.cs:line 110
> [...]

The error does not happen when I run my application as x86.

**Comment by 0rvar** [comment]
Thank you for opening an issue!

I could reproduce the issue. I have created a fix for this that I will be pushing momentarily, where both the 32bit and 64bit versions of the unmanaged DLL are included in the .NET DLL. I have tested running the test program on both 64 bit and 32 bit targets, and it seems to work.

I'll reply and close this once I have pushed and published the new version of the NuGet package.
